templates/entry/full.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Full entry template.
+ *
+ * @package AMPConf
+ */
+
+?>
+<article id="post-<?php the_ID(); ?>" <?php post_class( 'entry entry--featured' ); ?>>
+	<figure class="entry__thumbnail">
+		<a href="<?php esc_url( the_permalink() ); ?>">
+			<?php the_post_thumbnail( 'ampconf-375x225' ); ?>
+			<?php the_post_thumbnail( 'ampconf-1040x400' ); ?>
```

Comment:
Why calling `the_post_thumbnail()` twice? I'm seeing:

![image](https://user-images.githubusercontent.com/134745/35584750-c65d9282-05aa-11e8-8358-e766627a2053.png)

Is this related to https://github.com/xwp/ampconf/issues/48?